App.tsx

```diff
@@ -18,6 +19,7 @@ YellowBox.ignoreWarnings(['VirtualizedLists should never be nested inside', `\`-
 if (process.env.NODE_ENV !== 'development') {
   Sentry.init({
     dsn: 'https://dc67fd6d5c2949f2a6853e60eb69d899@o429172.ingest.sentry.io/5375289',
```

Comment:
I think we shouldn't keep URLs like this in the codebase. Not sure how it solved so far but I'd recommend using an env variable here and configure Bitrise in the way to build the app with the env

Reply:
yes, we have this sorted out on 3keys (develop), unfortunately, the 1key (master) doesn't have the possibility for configuring env as we added it later